pkg/trace/api/profiles_test.go

```diff
@@ -156,9 +162,16 @@ func TestProfileProxyHandler(t *testing.T) {
 	t.Run("ok", func(t *testing.T) {
 		var called bool
 		srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-			if v := req.Header.Get("X-Datadog-Additional-Tags"); v != "host:myhost,default_env:none" {
-				t.Fatalf("invalid X-Datadog-Additional-Tags header: %q", v)
+			v := req.Header.Get("X-Datadog-Additional-Tags")
+			tags := strings.Split(v, ",")
+			m := make(map[string]string)
+			for _, t := range tags {
+				kv := strings.Split(t, ":")
+				m[kv[0]] = kv[1]
 			}
+			containsTagHelper(t, m, v, "host")
+			containsTagHelper(t, m, v, "default_env")
+			containsTagHelper(t, m, v, "agent_version")
```

Comment:
Do you really need a function for this? You could just do:
```suggestion
for _, tag := range []string{"host", "default_env", "agent_version}{
	if _, ok := m[tag]; !ok {
		t.Fatalf("invalid X-Datadog-Additional-Tags header, should contain '%s': %q", tag, header)
	}
}
```

Reply:
Thanks, this is a good idea :)